node/src/ui_gateway/mod.rs

```diff
@@ -328,9 +336,15 @@ mod tests {
         system.run();
         let random_actor_recording = accountant_recording_arc.lock().unwrap();
         assert_eq!(random_actor_recording.len(), 0);
-        TestLogHandler::new().exists_log_containing(
-            "WARN: UiGateway: Deserialization error: expected value at line 1 column 1",
-        );
+        let expected_msg_example = "some bad bite for a jason processor; filling up to 120 characters: abcdefghijklmnopqrstuvwxyzabcdefg";
```

Comment:
Some of this following code might be simpler if you used

let mut expected_msg_example = payload();
expected_msg_example.truncate(100);